src/App.vue

```diff
@@ -1,48 +1,60 @@
 <template>
+
   <v-app>
     <v-app-bar app>
       <v-toolbar-title><b>Karina</b> - The Kubernetes Dashboard</v-toolbar-title>
     </v-app-bar>
 
     <v-main>
       <v-container fluid>
-        <v-row>
+  
+          <v-row>
           <v-col
             cols="12" xs="12" sm="12" md="6" lg="6" xl="6"
             v-for="(cluster, i) in clusters"
             :key="i"
           >
+      
             <v-row no-gutters>
               <v-col cols="12" xs="12" sm="12" md="12" lg="12" xl="12">
                 <IndicatorPanel
                   :cluster="cluster"
                   :itemicons="cluster.itemicons"
                 />
+
               </v-col>
             </v-row>
-          </v-col>
-         
+          </v-col>         
         </v-row>
       </v-container>
     </v-main>
   </v-app>
+
 </template>
 
+
 <script>
 
   import IndicatorPanel from './components/IndicatorPanel.vue'
-  import stats from './data/sample.json'
 
   export default {
 
     components:{
-      IndicatorPanel
+      IndicatorPanel,
     },
 
     data () {
       return {
-        clusters: stats.clusters
+        clusters: null,
       }
+    },
+
+    mounted() {
+      this.$axios
```

Comment:
Support demo mode i.e. `if (DEMO_MODE) {this.clusters = demoData}`

Then in the index.html that gets deployed to netlify we set in the html
```html
<script>
var DEMO_MODE = true
</script>
```

And in the Docker image we  use a different index.html that doesn't have the DEMO_MODE enabled.

Reply:
ok, thank you